### DISCUSSION TITLE
Add Text To Alert

### DISCUSSION THREAD

**Discussion by texasbobs** [fluxcd/flux2#1803] [Q&A] [answered]
Is it possible to add information to an alert?  We are performing tests using the `generic` and the `slack` providers currently.  For the slack provider, I see it is possible to add the `summary` and some text.  This text appears in the slack message.  For the generic type, adding `summary` does not work.

So our problem is we have no idea from which of the 2000 clusters the alert belongs to.

**Comment by stefanprodan** [comment, marked as answer]
The summary should be included in the generic payload. Please open an issue in notification-controller repo. Thanks for reporting this.

**Comment by texasbobs** [comment]
My mistake.  We were not routing to the correct API.  Summary is indeed working.
